trestle/resources/config.ini

```diff
@@ -0,0 +1,27 @@
+[catalog]
+decomposition_rules = ['catalog.groups.*.controls.*']
+number_of_groups = 2
```

Comment:
Seem to have lost the comment here - do we want to change 'number_of_groups' to 'create_number_of_groups' or similar?

Reply:
done